app/templates/components/load-more.hbs

```diff
@@ -0,0 +1,20 @@
+<div>
+  {{!-- If missing notice text provided, automatically show/hide MissingNotice and yield --}}
+  {{#if this.missingNotice}}
+    {{#if this.isNoneFound}}
+      <MissingNotice @title={{this.missingNotice}} />
+    {{else}}
+      {{yield this}}
+    {{/if}}
+  {{!-- Otherwise, just yield and let them handle it if they want --}}
+  {{else}}
+    {{yield this}}
+  {{/if}}
```

Comment:
I'd like to suggest a little bit different approach here:
```hbs
  {{#if this.isNoneFound}}
    {{#if hasBlock}}
      {{yield this}}
    {{else}}
      <MissingNotice @title={{or (this.missingNotice 'Some default not found message')}} />
    {{/if}}
  {{/if}}
```
In this case when nothing's found it'll yield whatever is in the block if it `hasBlock` (built-in hbs helper), otherwise it'll show `missingNotice` or fallback to default message.
What do you think?

Reply:
Hmm this sounded good to me at first, but after thinking about it for a bit, I'm not sure. The `LoadMore` component is meant to be used in block form, so in this case would it always yield? Which actually is maybe fine, I'm now thinking maybe that whole bit should be replaced with just `{{yield this}}`. It would reduce the complexity of this component. And the component user is already responsible for looping through and displaying the items, so maybe it makes more sense anyway. WDYT?